go/consensus/cometbft/apps/roothash/messages.go

```diff
@@ -131,6 +131,12 @@ func (app *rootHashApplication) changeParameters(ctx *api.Context, msg interface
 	if err != nil {
 		return nil, fmt.Errorf("roothash: failed to load consensus parameters: %w", err)
 	}
+	var needToDeletePastRoots bool
```

Comment:
```suggestion
	oldMaxPastRootsStored := params.MaxPastRootsStored 
```
I think that this way you can get rid of 1 `if` and 1 comparison.

Reply:
I think this microoptimization is unnecessary, since the entire change parameters call is going to be slow anyway.

Reply:
OK, marking suggestion as resolved.
I'm aware that optimization is negligible, but less code sometimes improves readability.